docs/release-notes/NuGet-5.5.md

```diff
@@ -0,0 +1,106 @@
+
+* dotnet restore failing with TaskCanceledException - [#7842](https://github.com/NuGet/Home/issues/7842)
+
+* Lock file is not honored in "*" scenarios  - [#8073](https://github.com/NuGet/Home/issues/8073)
+
+* NuGet.exe does not resolve to the latest version of a package when using * in PackageReference (MSBuild/Dotnet/VS restore do) - [#8432](https://github.com/NuGet/Home/issues/8432)
+
+* dotnet list package with multi targeting WPF project - [#8463](https://github.com/NuGet/Home/issues/8463)
+
+* Plugin:  "A task was cancelled" - problem with ADO authentication due to this. - [#8528](https://github.com/NuGet/Home/issues/8528)
+
+* Improve ConcurrencyUtilities (reduce CPU usage) - [#8653](https://github.com/NuGet/Home/issues/8653)
+
+* DG Spec for unloaded project scenarios should not be written in preview restores - [#8793](https://github.com/NuGet/Home/issues/8793)
+
+* The Visual Studio NuGet packages (RestoreManagerPackage) needs to auto load on solution build events - [#8796](https://github.com/NuGet/Home/issues/8796)
+
+* Deadlock in VSSettings init - [#8842](https://github.com/NuGet/Home/issues/8842)
+
+* VisualStudio ToolBox is not populated from a NuGet package if a project is placed in a solution folder - [#8868](https://github.com/NuGet/Home/issues/8868)
+
+* VS:  solution restore perpetually fails due to race condition - [#8881](https://github.com/NuGet/Home/issues/8881)
+
+* Constant "loading.." on installed tab, and "searching <term>.." on updates tab - [#8890](https://github.com/NuGet/Home/issues/8890)
+
+* Accessibility Fixes in PM UI - altText, etc... - [#9059](https://github.com/NuGet/Home/issues/9059)
+
+* Missing Embedded Icons in VS PM UI after cache expires - [#9069](https://github.com/NuGet/Home/issues/9069)
+
+* Review Accessibility bugs - [#9077](https://github.com/NuGet/Home/issues/9077)
+
+* PMUI Tabs have Accessibility Issues - [#9078](https://github.com/NuGet/Home/issues/9078)
+
+* FireAndForget PM UI startup - [#9112](https://github.com/NuGet/Home/issues/9112)
+
+* Accessibility Fixes in PM UI - [#9157](https://github.com/NuGet/Home/issues/9157)
+
+* Restore: IncludeExcludeFiles.Equals(...) implementation is incorrect - [#9167](https://github.com/NuGet/Home/issues/9167)
+
+* Restore: PackageSpec.Clone() creates unequal clone - [#9211](https://github.com/NuGet/Home/issues/9211)
+
+* Error list shown although "Always show Error List if build finishes with errors" is not checked - [#8190](https://github.com/NuGet/Home/issues/8190)
+
+* Static Graph restore should not pass empty SolutionPath - [#9061](https://github.com/NuGet/Home/issues/9061)
+
+* Restore: closure computed for each project 4 times - [#9042](https://github.com/NuGet/Home/issues/9042)
+
+* Restore: DependencyGraphSpec.Load(...) does not need JObject - [#9040](https://github.com/NuGet/Home/issues/9040)
+
+* Restore: large strings created on large object heap (LOH) - [#9031](https://github.com/NuGet/Home/issues/9031)
```

Comment:
We need a link to GH as we have in previous rel notes
![image](https://user-images.githubusercontent.com/16904420/77204618-5ae58100-6ab0-11ea-8871-64b4e2fec8b3.png)

Reply:
added **[List of all issues fixed in this release - 5.5](https://app.zenhub.com/workspaces/nuget-client-team-55aec9a240305cf007585881/reports/release?release=5e0e5fbd021f7aa0ec95db18)** in the next commit